cluster/juju/layers/kubernetes-worker/reactive/kubernetes_worker.py

```diff
@@ -20,6 +20,7 @@
 import shutil
 import subprocess
 import time
+import json
```

Comment:
This is already imported on line 17:
```
$ flake8 kubernetes_worker.py 
kubernetes_worker.py:23:1: F811 redefinition of unused 'json' from line 17
```

Mind fixing that up? Our automation will fuss about it.

Reply:
Thanks for noticing that.